docs/spec_draft.md

```diff
@@ -1406,6 +1407,76 @@ and produces a `result` tensor. More formally,
 ```
 
 [Back to Ops](#index-of-ops)
+
+## stablehlo.rng
+
+### Semantics
+
+Constructs an output of a given shape with random numbers generated given
+`rng_distribution`.
+
+`rng_distribution` is one of the following:
```

Comment:
With the explanation above, we do not need any of the following (including the formalism).